src/index.html

```diff
@@ -11,7 +11,165 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="box">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+          type="text"
+          name="username"
+          autocomplete="off"
+          required
+          >
```

Comment:
Please, fix such indentation for all similar cases
```suggestion
          <input
            type="text"
            name="username"
            autocomplete="off"
            required
          >
```